docs/docs/glossary.md

```diff
@@ -128,15 +145,15 @@ A markup language that every web browser is able to understand. It stands for Hy
 
 ### Headless CMS
 
-A [CMS](#cms) that only handles the [back-end](#back-end) content management instead of handling both the back-end and [front-end](#front-end). This type of setup is also referred to as [Decoupled](#decoupled).
+A [CMS](#cms) that only handles the [backend](#backend) content management instead of handling both the back-end and [frontend](#frontend). This type of setup is also referred to as [Decoupled](#decoupled).
```

Comment:
```suggestion
A [CMS](#cms) that only handles the [backend](#backend) content management instead of handling both the backend and [frontend](#frontend). This type of setup is also referred to as [Decoupled](#decoupled).
```